tests/conftest.py

```diff
@@ -16,7 +16,7 @@ def init_app(app_dir):
 @pytest.fixture(scope="module")
 def app(tmpdir_factory):
     # create directory structure of project/meld/components
-    app_dir = tmpdir_factory.mktemp("project", numbered=False)
+    app_dir = tmpdir_factory.mktemp("project")
```

Comment:
if `numbered` is disabled, using the `app` fixture in multiple modules  crashes the tests